spaceship/lib/spaceship/tunes/app_version.rb

```diff
@@ -557,7 +557,10 @@ def setup_screenshots
 
       # generates the nested data structure to represent screenshots
       def setup_screenshots_for(row)
-        display_families = row.fetch("displayFamilies", {}).fetch("value", nil)
+        display_families_hash = row.fetch("displayFamilies", {})
```

Comment:
Yes, I tried this fix locally and it worked for me. However, your suggestion to change fallback type and check row for nil definitely make sense. Thanks!